var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -383,7 +388,7 @@ class Trilinos(CMakePackage, CudaPackage):
 
     # Variant requirements from packages
     depends_on('metis', when='+zoltan')
-    depends_on('libx11', when='+exodus')
+    depends_on('libx11', when='+exodus+libx11')
```

Comment:
This could be simplified to `when='+x11'` since the `conflicts` prevents `+x11~exodus`.